src/main/java/frc/JoystickBindings.java

```diff
@@ -0,0 +1,30 @@
+package frc;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+import frc.robot.Robot;
+import frc.joysticks.SmartJoystick;
+import frc.utils.utilcommands.InitExecuteCommand;
+
+public class JoystickBindings {
+
+	public enum JoystickBindSet {
+
+		NONE,
+		EMPTY,
+		SWERVE,
+		SECOND,
+		TEST;
+	}
```

Comment:
```suggestion
		TEST;

	}
```
Sorry

I think testing is better than test but do whatever the convention is in the code (if there is such). 
Also, what's the difference between None and Empty?